stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -679,16 +633,22 @@ public extension Tensor where Scalar : BinaryFloatingPoint {
   ///   - shape: The dimensions of the tensor.
   ///   - mean: The mean of the distribution.
   ///   - stddev: The standard deviation of the distribution.
-  ///   - state: The pseudorandom state in which the random numbers are being
-  ///     generated.
+  ///   - engine: The random engine which generates random bytes.
   ///
   @inlinable @inline(__always)
   init(randomNormal shape: TensorShape, mean: Scalar = 0, stddev: Scalar = 1,
-       state: RandomState? = nil) {
-    let uniform = Tensor(randomUniform: shape, state: state)
-    let boxMullerHelper = Tensor(randomUniform: shape, state: state)
-    let result = sqrt(-2 * log(uniform)) * cos(2 * .pi * boxMullerHelper)
-    self = result * stddev + mean
+       engine: RandomEngine? = nil) {
+    self = Tensor(
+      handle: _TFHoistable {
+        let engine = engine ?? ARC4RandomEngine.global
+        let dist = NormalFloatingPointDistribution<Scalar>(mean: mean, standardDeviation: stddev)
```

Comment:
Wrap to fit within 80 cols:
```swift
        let dist = NormalFloatingPointDistribution<Scalar>(
           mean: mean, standardDeviation: stddev
        )
```

Reply:
Done. All formatting should be good now. I had forgotten it was 80 not 100 chars.